tests/clients/digitalocean/test_digitaloceanclient.py

```diff
@@ -0,0 +1,208 @@
+import os
+import gevent
+import pytest
+import string
+from unittest import TestCase
+from jumpscale.loader import j
+
+
+class DigitalOcean(TestCase):
+    def setUp(self):
+        self.ssh_client_name = j.data.idgenerator.nfromchoices(10, string.ascii_letters)
+        self.dg_client_name = j.data.idgenerator.nfromchoices(10, string.ascii_letters)
+        ssh = j.clients.sshkey.get(name=self.ssh_client_name)
+
+        self.dg = j.clients.digitalocean.get(self.dg_client_name)
+        SSH_PRIVATE_KEY_PATH = os.getenv("SSH_PRIVATE_KEY_PATH")
```

Comment:
I know it's already merged, but for future updates, we should use lower case letters for local variables.